spec/components/lesson_group_component_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe LessonGroupComponent, type: :component do
+  it 'renders the list with a title' do
+    component = described_class.new(title: 'HTML Basics', lessons: [])
+
+    render_inline(component)
+
+    expect(page).to have_css('h3', text: 'HTML Basics')
+  end
+
+  it 'renders the list of lessons' do
+    count = rand(10)
+    component = described_class.new(
+      title: 'HTML Basics',
+      lessons: create_list(:lesson, count, title: 'Lesson title'),
+    )
+
+    render_inline(component)
+
+    expect(page).to have_css('div[data-test-id="lesson-list"]')
```

Comment:
Nit: I'd be tempted to drop this expectation. The other expectation tests the behaviour perfectly, and should should cover us better if the component wasn't rendering the lesson list during because of a refactor or some other reason.